### PR TITLE
TECH-2111 - Adding REFERRAL_SUBGRAPH_URL env vars

### PR DESCRIPTION
**Pull request by OleksandrUA**
# TECH-2111 - Adding REFERRAL_SUBGRAPH_URL env vars

## Changes 👷‍♀️
- Adding REFERRAL_SUBGRAPH_URL to Github Actions CICD pipelines
- Adding REFERRAL_SUBGRAPH_URL to Dockerfile as argument
- Both above to both staging and prod environments
  
## How to test 🧪
- Trigger Github Actions CICD pipeline to see if container is being built and deployed correctly

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -41,6 +41,7 @@ ARBITRUM_MAINNET_RPC_URL=""
 PRODUCT_HUB_KEY=
 ONE_INCH_API_KEY=""
 ONE_INCH_API_URL="https://api-oasis.1inch.io"
+REFERRAL_SUBGRAPH_URL=""
```
Comment (piotrwitek) [inline .env:44]: duplicated

**Review by piotrwitek — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (OleksandrUA) [reply .env:44]: @piotrwitek I don't think so. I can only see 1 `REFERRAL_SUBGRAPH_URL` in the `.env` file. Could you please point to second one?

Comment (piotrwitek) [reply .env:44]: please pull changes @OleksandrUA

Comment (OleksandrUA) [reply .env:44]: @piotrwitek done

**Review by piotrwitek — APPROVED** [review]

**Review by cristidas — APPROVED** [review]

**Pull request merged**